.travis.yml

```diff
@@ -1,8 +1,20 @@
-language: node_js
 sudo: false
+
+git:
+  depth: 1
+
+branches:
+  only:
+    - master
```

Comment:
I _think_ this will stop Greenkeeper from testing new versions of dependencies.

Something like `- /^greenkeeper.*$/` might be needed to fix it.